third_party/systemlibs/mkl_dnn.BUILD

```diff
@@ -0,0 +1,65 @@
+licenses(["notice"])  # BSD/MIT-like license
+
+filegroup(
+    name = "LICENSE",
+    visibility = ["//visibility:public"],
+)
+
+
+HEADERS = [
+  "dnnl_config.h",
+  "dnnl_debug.h",
+  "dnnl.h",
+  "dnnl.hpp",
+  "dnnl_types.h",
+  "dnnl_version.h",
```

Comment:
Where are all these dnn* headers from? The mkldnn package doesnt have them? eg: https://packages.ubuntu.com/eoan/amd64/libmkldnn-dev/filelist

Reply:
Ubuntu ships the deprecated names only. The dnnl variants are the modern headers.